frontend/src/components/QuestionInput/QuestionInput.tsx

```diff
@@ -171,8 +171,9 @@ export const FileAttachmentInput = ({ setFileBlobUrl }: { setFileBlobUrl: (url:
 
 export const QuestionInput = ({ onSend, disabled, placeholder, clearOnSend, extraButtonNewChat }: Props) => {
     const { user, organization } = useAppContext();
-
+    const [agent, setAgent] = useState<string>("");
```

Comment:
@ramirezmorac2 you don.t need this state you suppose to take this from the context